transport.go

```diff
@@ -116,7 +131,14 @@ func (t *Transport) Token(ctx context.Context) (string, error) {
 	if t.token == nil || t.token.ExpiresAt.Add(-time.Minute).Before(time.Now()) {
 		// Token is not set or expired/nearly expired, so refresh
 		if err := t.refreshToken(ctx); err != nil {
-			return "", fmt.Errorf("could not refresh installation id %v's token: %s", t.installationID, err)
+			errorMessage := fmt.Sprintf("could not refresh installation id %v's token: %s", t.installationID, err)
+			re, ok := err.(*RefreshTokenError)
+			if ok {
+				re.Message = errorMessage
+				return "", re
+			}
+
+			return "", errors.New(errorMessage)
```

Comment:
We should be able to lean into error wrapping (https://blog.golang.org/go1.13-errors) here to simplify things:

```suggestion
	return "", fmt.Errorf("could not refresh installation id %v's token: %w", t.installationID, err)
```

Reply:
TIL: %w for wrapping errors. Thanks!